person.rb

```diff
@@ -4,14 +4,15 @@
 require_relative 'rentable'
 
 class Person < Nameable
-  attr_accessor :name, :age, :rentals
+  attr_accessor :name, :age, :rentals, :parent_permission, :classroom
   attr_reader :id
 
-  def initialize(age, name = 'unknown', parent_permission: true)
+  def initialize(age, name, classroom, parent_permission: true)
```

Comment:
- [ ] I got the error message screenshot below when I wanted to create a teacher 👇 

<img width="583" alt="Untitled" src="https://github.com/codeWithHyder/my-school-library/assets/103386183/7828085e-f415-48b2-b69e-374f7b76210e">